Lib/fontTools/misc/xmlWriter.py

```diff
@@ -173,6 +175,30 @@ def escape(data):
     data = data.replace("<", "&lt;")
     data = data.replace(">", "&gt;")
     data = data.replace("\r", "&#13;")
+
+    log = logging.getLogger("fontTools.ttx")
```

Comment:
also getLogger() can be called once outside the function, usually at the top of the module